app/ui/src/main/java/com/fsck/k9/ui/settings/KoinModule.kt

```diff
@@ -1,8 +1,8 @@
 package com.fsck.k9.ui.settings
 
-import com.fsck.k9.ui.helper.FileBrowserHelper
 import com.fsck.k9.helper.NamedThreadFactory
 import com.fsck.k9.ui.account.AccountsLiveData
+import com.fsck.k9.ui.helper.FileBrowserHelper
```

Comment:
Can you please undo this change? It will most likely lead to a merge conflict with the minSdkVersion 19 pull request.

Reply:
Sure. Will do